tests/cli/conftest.py

```diff
@@ -22,10 +27,28 @@ def do_run(*args, stdin):
     return do_run
 
 
+@pytest.fixture(scope="session")
+def init_default_project(tmpdir_factory: TempdirFactory) -> str:
+    path = tmpdir_factory.mktemp("agent").strpath
+    os.environ["LOG_LEVEL"] = "ERROR"
+
+    check_call(["rasa", "init", "--no-prompt"], cwd=path)
+    return path
+
+
 @pytest.fixture
-def run_in_default_project(testdir: Testdir) -> Callable[..., RunResult]:
+def run_in_default_project(
+    testdir: Testdir, init_default_project: str
+) -> Callable[..., RunResult]:
     os.environ["LOG_LEVEL"] = "ERROR"
-    testdir.run("rasa", "init", "--no-prompt")
+
+    # makes sure we do not always retrain an initial model for every "new" project
+    for file_name in os.listdir(init_default_project):
+        full_file_name = os.path.join(init_default_project, file_name)
```

Comment:
can't we `copytree` the whole directory?

Reply:
optional: use `Path`

Reply:
copytree: no since the target dir already exists copytree would fail...